src/lib/holocene/link.svelte

```diff
@@ -32,8 +32,9 @@
   export let light = false;
 
   const onLinkClick = (e: MouseEvent) => {
-    // Skip if middle mouse click or new tab
-    if (e.button === 1 || newTab || e.metaKey) return;
+    if (e.button === 1 || newTab || e.metaKey || e.ctrlKey || e.shiftKey)
+      return;
+
     e.preventDefault();
     goto(href);
   };
```

Comment:
Remind me why we need to do any of this? Is it a remnant of some old url encoding stuff? Seems to me we should just remove the on:click handler from the anchor tag entirely.

Reply:
So we need this because otherwise the <a href...> would do a hard navigation and reload everything from the top level layout every time we navigate. So we need to use goto instead. But in the case of middle button click/newTab/meta/ctrl/shift, we don't want to use goto, but instead use the default broswer behavior of open in new tab/window

Reply:
But now that you say that, I just tested this with Svelte5 and i think its fixed

Reply:
Nevermind, we still need it

Reply:
Ah ok, makes sense. Maybe there's something [here](https://svelte.dev/docs/kit/load#Rerunning-load-functions) we can do, but this is fine for now